File: Back-End/ViewEvents.Domain/Models/Speaker.cs
namespace ViewEvents.Domain.Models
{
    public class Speaker
    {
        public int Id { get; set; }
        public string Name { get; set; }
        public string Summary { get; set; }
        public string ImgURL { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public IEnumerable<SocialNetwork> SocialNetworks { get; set; }
        public IEnumerable<EventSpeaker> EventSpeakers { get; set; }
    }
}

File: Front-End/ViewEvent.Front/src/app/models/speaker.ts
import { SocialNetwork } from "./social-network";
import { Event } from "./event";

export interface Speaker {
    id: number;
    name: string;
    summary: string;
    imgUrl: string;
    phoneNumber: string;
    email: string;
    socialNetworks: SocialNetwork[];
    eventSpeakers: Event[];
}
